ros/src/tl_detector/tl_detector.py

```diff
@@ -100,8 +139,32 @@ def get_closest_waypoint(self, pose):
             int: index of the closest waypoint in self.waypoints
 
         """
-        #TODO implement
-        return 0
+        wp_id = None
+        wps = waypoints.waypoints
+
+        x = pos_x
+        y = pos_y
```

Comment:
Do these need to be copied?

Reply:
I updated the algorithm after debug- but forgot to clean this up. Will update the branch with this fixed.